src/components/media-types/index.js

```diff
@@ -100,7 +100,7 @@ export const renderMediaType = ({
     case MIMETYPE.OGA:
       return (
         <Container interactive={interactive}>
-          <AudioComponent src={url} />
+          <AudioComponent src={url} name={metadata.token_info.name} />
```

Comment:
can you just pass the metadata like the vectorcomponent?

Reply:
sure!

Reply:
done